dist/filelist

```diff
@@ -50,6 +50,7 @@ src/btree/row_srch.c
 src/checksum/arm64/crc32-arm64.c	ARM64_HOST
 src/checksum/power8/crc32.sx		POWERPC_HOST
 src/checksum/power8/crc32_wrapper.c	POWERPC_HOST
+src/checksum/riscv64/crc32-riscv64.c	RISCV64_HOST
```

Comment:
I'd have preferred not to need to add this since we don't actually have hw acceleration on this platform (yet), but I couldn't figure out how to instruct it to do a graceful fallback to sw without adding such a file. Please let me know if there is another way.

Reply:
For lots of reasons (including that MongoDBb builds out the checksum code as a separate, non-WiredTiger library and gcc and clang have their own weird syntax requirements), this is just a configuration mess. Having a separate file is probably the cleanest thing to do and doesn't cost us any performance, I think it's the right choice and avoids file shuffling when we add acceleration.